lib/rules/keyframe-selector-notation/__tests__/index.js

```diff
@@ -301,6 +341,26 @@ testRule({
 		{
 			code: '@keyframes foo { from,to {} }',
 		},
+		{
```

Comment:
suggestion

put these in a const and destructure it 3 times

Reply:
Thanks, I agree with your suggestion. https://github.com/stylelint/stylelint/pull/6605/commits/ca57baf4ed0404f8202514eff9e19e5bd7cca5ae